File: 30 Days of Code/Day 5 Loops/Solution.py
# https://www.hackerrank.com/challenges/30-loops/problem

#!/bin/python3

import math
import os
import random
import re
import sys

def printMultiples(number):
    for i in range(1,11):
        print(str(number)+" x "+str(i)+" = "+str(number*i))

if __name__ == '__main__':
    n = int(input())
    printMultiples(n)

File: 30 Days of Code/Day 5 Loops/Solution.cpp
// https://www.hackerrank.com/challenges/30-loops/problem

#include <bits/stdc++.h>

using namespace std;

void findMultiples(int number){
    for(int val = 1; val <= 10; ++val){
        cout<<number<<" x "<<val<<" = "<<number*val<<endl;
    }
}

int main()
{
    int n;
    cin >> n;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');
    findMultiples(n);
    return 0;
}
